src/color-picker/README.md

```diff
@@ -89,12 +112,20 @@ All attributes are reactive:
 | `space` | `space` | `ColorSpace` &#124; `string` | `oklch` | The color space to use for interpolation. |
 | `color` | `color` | `Color` &#124; `string` | `oklch(50% 50% 180)` | The current color value. |
 
+### Getters
+
+These properties are read-only.
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `spaceResolved` | `ColorSpace` | Color space object corresponding to the `space` property. |
```

Comment:
Given that in JS, `spaceResolved` is far more useful than `space`, I wonder if we should make `space` the one with the actual color space (and make it read/write), and `spaceRaw` or something the one that reflects. Is there any precedent in HTML where the object prop is the one with the "nicer" name?